9jaswag/PostIt
# #152632081 Adhere to Redux Architecture

#### What does this PR do?
- ensures actions dispatch data to the redux store

#### Description of Task to be completed?
- adhere to redux architecture

#### How should this be manually tested?
- check the ```client/reducers``` directory to see the new reducers that handle te store update

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- #152632081

#### Screenshots (if appropriate)
- None

#### Questions:
- None
